build.sbt

```diff
@@ -29,7 +29,8 @@ lazy val microservice = Project(appName, file("."))
       "models.Binders._",
       "java.time.OffsetDateTime",
       "v2.models.EORINumber",
-      "v2.models.MovementId"
+      "v2.models.MovementId",
+      "v2.models.MovementType"
```

Comment:
No need to add here. I have done it without adding ``` MovementType ``` and seems to be working.

Reply:
Correct, but the code for the binder is in the wrong place (it's currently in `models.Binders`, should really be in a `v2.models.Binders`.

I was a bit confused as to why that worked.